docs/content/en/docs/concepts/events.md

```diff
@@ -0,0 +1,190 @@
+          "name": "spaceship",
+          "image": {
+            "id": "docker.io/tgraf/netperf@sha256:8e86f744bfea165fd4ce68caa05abc96500f40130b857773186401926af7e9e6",
+            "name": "docker.io/tgraf/netperf:latest"
+          },
+          "start_time": "2023-10-06T21:52:41Z",
+          "pid": 49
+        },
+        "pod_labels": {
+          "app.kubernetes.io/name": "xwing",
+          "class": "xwing",
+          "org": "alliance"
+        },
+        "workload": "xwing"
+      },
+      "docker": "551e161c47d8ff0eb665438a7bcd5b4",
+      "parent_exec_id": "Z2tlLWpvaG4tNjMyLWRlZmF1bHQtcG9vbC03MDQxY2FjMC05czk1OjEzNTQ4NjQ1MjQ1ODM5OjUyNjg5",
+      "tid": 52699
+    }
+  },
+  "node_name": "gke-john-632-default-pool-7041cac0-9s95",
+  "time": "2023-10-06T22:03:57.700326678Z"
+}
+
+```
+</p>
+</details>
+
+Will only highlight a few important fields here. For a full specification of events see the Reference
+section. All events in Tetragon contain a `process_exec` block to identify the process generating the
+event. For execution events this is the primary block. For [Tracing Policy](({< ref >})) events the
+hook that generated the event will attach further data to this. The `process_exec` event provides
+a cluster wide unique id the `process_exec.exec_id` for this process along with the metadata expected
+in a Kubernetes cluster  `process_exec.process.pod`. The binary and args being executed are part of 
+the event here `process_exec.process.binary` and `process_exec.process.args`. Finarlly a `node_name`
+and `time` provide the location and time for the event and will be present in all event types.
+
+A default deployment writes the JSON log to `/var/run/cilium/tetragon/tetragon.log` where it can
+be exported through normal log collection tooling, e.g. 'fluentd', logstash, etc.. The file will
+be rotated and compressed by default. See [Helm Options] for details on how to customize this location.
+
+#### `tetra` CLI
+
+A second way is to use the [`tetra`](https://github.com/cilium/tetragon/tree/main/cmd/tetra) CLI. This
+has the advantage that it can also be used to filter and pretty print the output. The tool
+allows filtering by process, pod, and other fields.
+
+If you are using `homebrew`, you can install the latest release with:
+
+```shell
+brew install tetra
+```
+
+Or you can download and install the latest release with the following commands:
+
+```shell
+GOOS=$(go env GOOS)
+GOARCH=$(go env GOARCH)
+curl -L --remote-name-all https://github.com/cilium/tetragon/releases/latest/download/tetra-${GOOS}-${GOARCH}.tar.gz{,.sha256sum}
+sha256sum --check tetra-${GOOS}-${GOARCH}.tar.gz.sha256sum
+sudo tar -C /usr/local/bin -xzvf tetra-${GOOS}-${GOARCH}.tar.gz
+rm tetra-${GOOS}-${GOARCH}.tar.gz{,.sha256sum}
+```
+
+See the [latest release
+page](https://github.com/cilium/tetragon/releases/latest) for supported
+`GOOS`/`GOARCH` binary releases.
```

Comment:
I think we should remove that part and link to the install guide for tetra CLI directly. It's duplicated info